src/Storages/ObjectStorage/StorageObjectStorageStableTaskDistributor.cpp

```diff
@@ -179,4 +199,28 @@ std::optional<String> StorageObjectStorageStableTaskDistributor::getAnyUnprocess
     return std::nullopt;
 }
 
+void StorageObjectStorageStableTaskDistributor::rerunTasksForReplica(size_t number_of_current_replica)
```

Comment:
If I understand correctly, this method re-schedules the tasks FROM a given replica to another one. Is that correct?

If that's so, I would name it differently:

`rescheduleTasksFromReplica`

Reply:
Yes, it adds tasks from replica to unprocessed list to allow other replicas take them.